src/Statamic.php

```diff
@@ -394,8 +394,9 @@ private static function createVersionedAssetPath($name, $path, $extension)
             return (string) $path;
         }
 
-        return Cache::rememberForever("statamic-{$extension}-{$name}", function () use ($path, $extension) {
+        $cachePath = substr($path, -40);
```

Comment:
Can you explain what the point of this substring is? What's 40?

Reply:
You're right. That's a little mysterious. 

It will make sure to limit the string to 40 characters, in case it's longer. The minus does say that it will keep the _last_ 40 characters, to always include the filename.

I added it that way, to avoid too long key names for caching, as there are limits. I don't think we would normally hit a limit, but this felt like a saver approach in case the path would be too loooooooooooooooooooooooooooooooooooooong.

If you want to keep it, I am happy to add a comment.

Reply:
I understand, thanks.